### DISCUSSION TITLE
Issue with props passed from ```getStaticProps```

### DISCUSSION THREAD

**Discussion by Mallikarjunamatla** [vercel/next.js#44896] [Help]
I have a React app as Host and Nextjs app as Remote , But problem is while running two builds props/data from ```getStaticProps``` is erased in React app.
this is the example provided : https://github.com/module-federation/module-federation-examples/tree/master/react-nextjs/react-host-nextjs-remote

**Comment by stevebrowndotco** [comment]
@Mallikarjunamatla I believe this happens because _app.tsx is unknown to the host. NextJs uses this as an entry point to pass in the server side props. I think this is solved with the new experimental app directory, but importing a module with this new approach seems to be unsupported
